networkx/algorithms/smallworld.py

```diff
@@ -354,15 +353,31 @@ def omega(G, niter=100, nrand=10, seed=None):
     # Compute the mean clustering coefficient and average shortest path length
     # for an equivalent random graph
     randMetrics = {"C": [], "L": []}
-    for i in range(nrand):
-        Gr = random_reference(G, niter=niter, seed=seed)
-        Gl = lattice_reference(G, niter=niter, seed=seed)
-        randMetrics["C"].append(nx.transitivity(Gl))
+
+    # Calculate initial average clustering coefficient which potentially will
+    # get replaced by higher clustering coefficients from generated lattice
+    # reference graphs
+    Cl = nx.average_clustering(G)
+
+    niter_lattice_reference = niter
```

Comment:
The niter_lattice_reference=5 default is taken from the upon based paper. As this takes significantly longer to calculate as the random reference, I increased the niter_random_reference to be closer to the previous default. In my opinion there is no natural to choose an appropriate value, I just chose the ones which where computationally feasible for some testes networks. Setting both niters to the same value might be fine too.